admin/app/tools/CloudWatch.scala

```diff
@@ -90,7 +90,7 @@ object CloudWatch extends Logging {
     "head.content.css",
     "head.facia.css",
     "head.football.css",
-    "head.identity.css",
```

Comment:
out of curiosity, what is this list used for?

Reply:
It looks like it's used for tracking bundle sizes in Frontend Admin dashboards.